lib/nxt_state_machine/errors/error.rb

```diff
@@ -1,3 +1,8 @@
 module NxtStateMachine
   Error = Class.new(StandardError)
```

Comment:
I think we can probably remove this too then 
```suggestion
```

Reply:
I thought we could but that is still needed.  It defines `NxtStateMachine::Error` and something depends on it (at least a test fails without it).